tracks/rust/exercises/clock/mentoring.md

```diff
@@ -7,7 +7,14 @@
 ### Reasonable solutions
 
 A reasonable solution should do the following:
```

Comment:
Let's amend it to say:

> A reasonable solution should:
> 
> 	- Use `#[derive]`
> 	...

Also, I feel it reads most straightforward to use `1440` as the value for the constant, because the variable name `MINUTES_PER_DAY` tells us exactly what we're looking at. Also, having it set to a plain integer is "simpler" because there are no maths that need to be evaluated.